pkg/destroy/registry/registry.go

```diff
@@ -0,0 +1,19 @@
+package registry
```

Comment:
can we name it backends/providers etc...? registry seems a little odd.

Reply:
ack, fixed